org.uqbar.project.wollok.lib/src/wollok/lib.wlk

```diff
@@ -234,6 +235,12 @@ class Position {
 	
 	method clone() = new Position(x, y)
 
+	method distance(position) {
+	    const deltaX = self.getX() - position.getX()
+	    const deltaY = self.getY() - position.getY()
+	    return (deltaX.square() + deltaY.square()).squareRoot() 
+	}
```

Comment:
This impl is fine. Just thinking that we could have more methods working with tuples / vectors.
like
```
method distance(position) {
   (self.position() - position).map { coord => coord.square }.fold {a, b => a +b }.squareRoot()
}```

WIth "it" operator this could be

method